kmath-units/src/commonMain/kotlin/space/kscience/kmath/units/units.kt

```diff
@@ -0,0 +1,276 @@
+package space.kscience.kmath.units
+
+import space.kscience.kmath.operations.*
+import kotlin.jvm.JvmName
+import kotlin.math.PI
+import kotlin.math.pow
+
+/**
+ * Represents base units of International System of Units.
+ */
+public enum class BaseUnits {
```

Comment:
Enums are not very extensible, so if the library will lack some base units, users will no have a choice rather than wait for an update in the library. What about converting `BaseUnits`  into an interface, so any library client will be able to define their own base units, like:
```
object Bit : BaseUnit

public val bit: Measure = Measure(Bit to 1)
public val bitrate: Measure = MeasureAlgebra { bit / s }
```